Tutorial.md

```diff
@@ -1,7 +1,8 @@
-# API Blueprint Tutorial
-Welcome to the API Blueprint Tutorial! In this tutorial you will learn all about all the key features of the API Blueprint Language on the example of an API for our imaginary industry-changing Gist service.
+# API Blueprint Basic Tutorial
 
-Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set some expectations let's provide following functionality in our API: *List Gists, Retrieve a Single Gist, Create a Gist, Edit a Gist, Star a Gist, Unstar a Gist, Check if a Gist is starred and Delete a Gist.*
+Welcome to the API Blueprint Tutorial! In this tutorial you will learn about the basics to getting started with the API Blueprint language.
+
+We’re going to build an API Blueprint for a service called polls, a simple web service that allows consumers to view polls and vote in them.
```

Comment:
Again sorry for possibly being illiterate but should Polls be with capital P? "a service called polls" also I feel the dash is more appropriate "a service called Polls – a simple web service.."